src/schema/nmdc.yaml

```
@@ -981,6 +981,14 @@ enums:
       aqueous_layer:
       non_polar_layer:
 
+  UnitEnum:
+    permissible_values:
+      Cel:
+        aliases:
+          - Celsius
```

```suggestion
          - Celsius
          - C
          - degreeCelsius
          - degree Celsius
```
I am not sure LinkML will handle the special character for `°` correctly, so I have not included it here; however, I think it would be another good alias to capture. I will test with `°C` separately from this PR.